data/journal_check/bug_refs.json

```diff
@@ -307,11 +307,17 @@
         "products": {
             "sle-micro": [ "5.0", "5.1", "5.2", "5.3" ],
             "leap-micro": ["5.2", "5.3"],
-            "microos":  ["Tumbleweed"],
```

Comment:
Yaiks - That makes all stagings fails from the time this was merged until the fixed health-checker is all around

Reply:
True. I'll leave the other active temporarily.